server.ts

```diff
@@ -88,6 +89,7 @@ export function app(): express.Express {
           // Both video services' iframe embeds seem to need script access to not error with our current embed approach.
           'https://player.vimeo.com',
           'https://www.youtube.com/',
+          'https://www.youtube-nocookie.com',
```

Comment:
Do we know that script access is needed for our own app? From https://makersaid.com/embed-youtube-without-cookies/ and the principle of the frame being more isolated for privacy reasons, I was hoping not.

Reply:
No, apparently the script access isn't needed, I just tested on local. Removed in 681a61c0